modules/api/src/main/java/org/apache/ignite/configuration/schemas/table/TableConfigurationSchema.java

```diff
@@ -15,23 +15,21 @@
  * limitations under the License.
  */
 
-package org.apache.ignite.configuration.extended;
+package org.apache.ignite.configuration.schemas.table;
 
 import org.apache.ignite.configuration.annotation.Config;
 import org.apache.ignite.configuration.annotation.Value;
+import org.apache.ignite.configuration.validation.Immutable;
 
-/** */
 @Config
-public class DataStorageConfigurationSchema {
-    /** */
+public class TableConfigurationSchema {
     @Value
-    public int pageSize;
+    @Immutable
+    public String name;
 
-    /** */
     @Value
-    public String storagePath;
+    public int partitions;
```

Comment:
Should we have validation annotations and default value here?

Reply:
Added limitations.